src/app/pages/privacy-policy/privacy-policy.component.scss

```diff
@@ -0,0 +1,35 @@
+@import "../../../variables";
+
+.home-container {
+  height: 100%;
+  display: grid;
+  place-items: center;
+  grid-template-rows: 1fr auto;
+  grid-template-columns: 100%;
+  margin-left: 8px;
+  margin-right: 8px;
+  overflow-y: scroll;
```

Comment:
It turns out this way of making the foreground scroll doesn't really work well and especially makes it feel bad on desktop. See some of the changes we made in the about page branch, namely the CSS changes in this commit: [`5417637` (#137)](https://github.com/mn-pollinators/buzz-about/pull/137/commits/5417637b57c44abb6f5398d481daf438291f4cac).